### DISCUSSION TITLE
Chia support

### DISCUSSION THREAD

**Discussion by janwarming** [LedgerHQ/ledger-live-desktop#3990] [Coin/Token request]
We want chia support in ledger live ;)

**Comment by adam8797** [comment]
+1 but I think this would also require a whole chia app to be developed for the ledger itself.

**Comment by jack60612** [comment]
+1

**Comment by tuanson242** [comment]
+1

**Comment by johodh** [comment]
+1

**Comment by Gregwolf3030** [comment]
Agreed. All my crypto live in Ledger. Need a secure place for my Chia.

**Comment by marceus** [comment]
+1

**Comment by Studmeister247** [comment]
+1

**Comment by trondve** [comment]
+1

**Comment by Pockles** [comment]
+1, give me a reason to use my ledger that's still sitting in its box!

**Comment by jon-peterson** [comment]
+1

**Comment by dhatzenbichler** [comment]
+1

**Comment by ioanvm** [comment]
+1

**Comment by butterisgod** [comment]
This issue is really important. I know the dev team is busy, but please make this a priority!!

**Comment by jack60612** [comment]
@gre when is chia support being added as the demand is growing everyday  @rjarasson-ledger

**Comment by liquidtroll** [comment]
+1 we need this feature asap

**Comment by 4jin** [comment]
+1

**Comment by Hawstly** [comment]
+1 deweet

**Comment by Havoc4Hire** [comment]
+1

**Comment by wishwanto** [comment]
+1

**Comment by billcod** [comment]
Agreed, but I would also like Trezor Support.

**Comment by jack60612** [comment]
> Agreed, but I would also like Trezor Support.

This is not for trezor.
Also the trezor is not secure at all

**Comment by l4z4ru53** [comment]
+1

**Comment by busigkille** [comment]
+1

**Comment by hawaii883** [comment]
+1

**Comment by Noob-granit** [comment]
+1

**Comment by abb0r** [comment]
+1

**Comment by seertenedos** [comment]
Even if this could just securly generate public keys to send to for now that would be great.  Currently you cna only generate a public key by loading the private key onto a machine

**Comment by jimkoen** [comment]
+1

**Comment by pablopgus** [comment]
+1

**Comment by toniodlab** [comment]
+1 Chia support would be very useful

**Comment by jinthagerman** [comment]
+1

**Comment by heidris** [comment]
+1

**Comment by kwoods7** [comment]
+1

**Comment by mmoadvert** [comment]
+1 (make chia support in ledger live)

**Comment by jimkoen** [comment]
+1 Just give us BLS Signature Support and I'm sure some community members could cook something up

FYI i don't currently own a ledger wallet, but this would be a major incentive to buy one. I'm happy to pay between 100-200 USD for decent chia integration, hell, give me a chia branded one if thats more profitable for you.

**Comment by DCYeahThatsMe** [comment]
+1 absolutely would love to have my Nano S support CHIA! It would even be reason to buy another or upgrade to the X!

**Comment by eric-vader** [comment]
+1 Would like this

**Comment by dustingrady** [comment]
+1 would love to see this added in the near future!

**Comment by Ealrann** [comment]
+1

**Comment by namen3645** [comment]
+1

**Comment by Phoenix603** [comment]
+1

**Comment by blendedw** [comment]
+1

**Comment by kevin-sorat** [comment]
+1
